features/branch.feature

```diff
@@ -0,0 +1,39 @@
+@ruby>=2.1
+Feature: Check out a feature branch based on a story from Pivotal Tracker
+  Background:
+    Given a file named "tmp/global_settings.yml" with "pivotal_tracker_api_key: my_api_key"
+
+  Scenario: Checkout a new branch with geordi branch
+    Given my username from git config is "Max Musterman"
+      And my local git branches are: master
+
+    When I run `geordi branch` interactively
+      # I skip the initials prompt
+      And I type ""
+    Then the output should contain "Util.run! git, checkout, master"
+      And the output should contain "Util.run! git, checkout, -b, mm/test-story-12"
```

Comment:
I really don´t know why this is happening. 
Here it is the same: 
https://github.com/makandra/geordi/blob/2234586f7a3f88ffc52fd9d6f455e60f688b8127/features/commit.feature#L11
I dont thinks I am running my tests wrong, because it also has this behavior in the github checks.
Maybe some ruby require magic that is wrong in `gitpt.rb`?

Reply:
Ok sorry, it was me. Tests print commands with commas https://github.com/makandra/geordi/blob/2234586f7a3f88ffc52fd9d6f455e60f688b8127/lib/geordi/util.rb#L61-L64